addons/hw_drivers/tools/helpers.py

```diff
@@ -26,6 +28,7 @@
 from odoo.tools.misc import file_path
 
 _logger = logging.getLogger(__name__)
+ODOO_CONF = '/home/pi/odoo/addons/point_of_sale/tools/posbox/configuration/odoo.conf'
```

Comment:
RIP windows IoT

Reply:
would it be ok like this: `ODOO_CONF = file_path('odoo/addons/point_of_sale/tools/posbox/configuration/odoo.conf')` ?

Reply:
nope. For windows IoT, the .conf file is located in the same one than the odoo's code. In fact, it does totally ignore the `/point_of_sale/tools/posbox/configuration/odoo.conf` file at all. It just create the default one then manually modify the config file with what is needed for IoT, see:
https://github.com/odoo/odoo/blob/17.0/setup/win32/setup.nsi#L307-L309

I think it would be safer to use something like `from odoo import tools; tools.config.config_file`. But, this would only work when odoo is ran as a service (not manually in the cli) so might be a good idea to have a failsafe mechanism

Reply:
@loouis-t looks like the file path might be in `tools.config.rcfile` but it can indeed be a good idea to fallback on the default paths if the file have not been set in the CLI.
Or maybe consider using it's own config file rather that will be on a known location instead of relying on the one of odoo
![image](https://github.com/user-attachments/assets/8edf3645-e05d-4d50-b02a-ba721c7fdf29)

Reply:
problem is that calling `tools.config` triggers the `__init__.py` method from [odoo/tools/config.py](https://github.com/odoo/odoo/blob/ca7c212cca50f931ad5be7f74311723fe56d08e5/odoo/tools/config.py#L65) that directly adds configurations in `odoo.conf` that are unwanted for the iot. I'd say we keep the hardcoded path.